index.html

```diff
@@ -2,7 +2,10 @@
 <html>
     <head>
```

Comment:
Some indentation to make the file prettier, and easy to read

Reply:
Fixed with 8ff662616f76f10c7d0c72a94a31d1216c4439f8